specification/_types/Geo.ts

```diff
@@ -82,3 +82,60 @@ export class LatLon {
   lat: double
   lon: double
 }
+
+/**
+ * A latitude/longitude as a 2 dimensional point. It can be represented in various ways:
+ * - as a `{lat, long}` object
+ * - as a geo hash value
+ * - as a `[lon, lat]` array
+ * - as a string in `"<lat>, <lon>"` or WKT point formats
+ *
+ * @codegen_names latlon, geohash, coords, text
```

Comment:
minor nit: can we use `coordinates` instead of `coords`?

Reply:
As discussed, `coords` is heavily used in the geo domain, so it keeps things short without hurting understandability.